deposit_service_test.go

```diff
@@ -22,7 +22,8 @@ func (s *depositServiceTestSuite) TestListDeposits() {
                 "insertTime": 1508198532000,
                 "amount": 0.04670582,
                 "asset": "ETH",
-                "status": 1
+                "status": 1,
+                "TxID": "TxID"
```

Comment:
the expected txId should be `"txId": "b3c6219639c8ae3f9cf010cdc24fw7f7yt8j1e063f9b4bd1a05cb44c4b6e2509",` as in https://github.com/binance-exchange/binance-official-api-docs/blob/master/wapi-api.md#deposit-history-user_data